plugins/sms-twilio/src/src/Plugin.php

```diff
@@ -71,37 +75,48 @@ private function processCli(): void
         if ($this->pluginDataValidator->validate()) {
             $this->logger->info('Validating config');
             $this->optionsManager->load();
-            $this->logger->info('CLI process ended.');
         }
     }
 
     private function processHttpRequest(): void
     {
+        $pluginData = $this->optionsManager->load();
+        if ($pluginData->logging_level) {
+            $this->logger->setLogLevelThreshold(LogLevel::DEBUG);
```

Comment:
also default threshold should be INFO

Reply:
It is - that's why we set it here explicitly.